tests/OSPSuite.Starter/Bootstrapping/ApplicationStartup.cs

```diff
@@ -46,7 +46,7 @@ public class ApplicationStartup
       public static void Initialize()
       {
          initializeDependency();
-         fillDimensions(IoC.Resolve<IDimensionFactory>());
+         fillDimensions(IoC.Container.Resolve<IDimensionFactory>());
```

Comment:
Why Container?? Does not matter but this should not be required

Reply:
You are right... I have just removed it...